docs/chofmajp.md

```diff
@@ -5,3 +5,5 @@
 **Occupation:** Engineer
 
 **Location:** Milwaukee
+
+**Favorite Vacation Spot:** Michigan
```

Comment:
Is there a particular city in Michigan?  Or just Michigan in general?  The whole state is kind of :sparkles:amazing:sparkles:.